test/utils/validate-base.test.ts

```diff
@@ -45,4 +45,13 @@ describe("validation: validateBase", () => {
       "This remark was issued under version 0.0.0 instead of 1.0.0"
     );
   });
+
+  it("should throw - if space is present", () => {
```

Comment:
Is this right? It tests for RMRK version but should throw if space is present?

Reply:
Removed now